app/src/main/java/org/mozilla/focus/shortcut/IconGenerator.kt

```diff
@@ -23,37 +23,55 @@ class IconGenerator {
 
     companion object {
         private val TEXT_SIZE_DP = 36f
-        private val DEFAULT_ICON_CHAR = "?"
+        private val DEFAULT_ICON_CHAR = '?'
 
         /**
-         * Use the given raw website icon and generate a launcher icon from it. If the given icon is null
-         * or too small then an icon will be generated based on the website's URL. The icon will be drawn
+         * Generate an icon based on the website's URL. The icon will be drawn
```

Comment:
nit: We should say, "See [generateLaunchIcon...] for more details" rather than duplicating the comment.